### ISSUE TITLE
Leds not working when connected via Bluetooth

### ISSUE THREAD

**Issue by Ctrl-F4** [kamaaina/macropad_tool#26]
Programmed the leds of my 12 buttons and 2 knobs keyboard and the keys blink when buttons are pressed. Very nice!
Leds are only blinking when the keyboard is powered via USB. (Connected via bluetooth only the leds stay off)
Is it possible to implement leds functioning when configured index 2/3/4 and connected via bluetooth?
Except when configured as backlight always on to prevent drain (index 1/5)
Or is this only possible by some hardware modification?

**Comment by Ctrl-F4** [comment]
Device: `USB\VID_1189&PID_8842\`
Yes referring to the index argument passing to the led command.
Don't know about the manufacturers software yet (don't like virus or malware) i'm glad i found your software instead.

I found [this post](https://github.com/kriomant/ch57x-keyboard-tool/issues/60#issuecomment-2200715830) later , someone with same "issue", he "solved" it by using a powerbank as a workaround.
So next step is to find out where to download the manufacturers software... 
When i found out i'll report back later (but maybe someone already knows, feel free to tell in the meantime)

**Comment by kamaaina** [comment]
https://github.com/kamaaina/macropad_tool/issues/27 might be related, but as i commented there, i do not have a bluetooth macropad to test/implement changes on. 

are you able to provide a usb capture from the manufacturer software? if so, i should be able to fix this issue

**Comment by spikerguy** [comment]
> I found [this post](https://github.com/kriomant/ch57x-keyboard-tool/issues/60#issuecomment-2200715830) later , someone with same "issue", he "solved" it by using a powerbank as a workaround.

All Macropad with LED can only power on the LED when power is supplied as the tiny battery inside the controller can only power the chip.

We should close this issue as LED only work when power is connected. I do exactly the same thing. I connect powerbank when I want to have led on.

**Comment by alternistom** [comment]
Yes even the macropad's manual mentions that the LED won't work on BT connection, so it's not related to this repo.
